src/main/java/org/spongepowered/api/item/ItemType.java

```diff
@@ -37,7 +39,7 @@
  * A type of item.
  */
 @CatalogedBy(ItemTypes.class)
-public interface ItemType extends CatalogType, Translatable {
+public interface ItemType extends CatalogType, Translatable, GameDictionaryEntry {
```

Comment:
Should `ItemType` implement `GameDictionaryEntry` or just provide a method for it?